test/built-ins/Function/prototype/caller-arguments/accessor-properties.js

```diff
@@ -0,0 +1,18 @@
+/*---
+description: Function.prototype caller and arguments properties are accessor properties with ThrowTypeError
+esid: sec-function.prototype.caller
+info: |
+    Function instances do not inherit the "caller" and "arguments" accessors
+    from Function.prototype. The accessors exist only on Function.prototype.
+---*/
+
+const callerDesc = Object.getOwnPropertyDescriptor(Function.prototype, "caller");
+const argumentsDesc = Object.getOwnPropertyDescriptor(Function.prototype, "arguments");
+
+assert.sameValue(typeof callerDesc.get, "function");
+assert.sameValue(typeof callerDesc.set, "function");
+assert.sameValue(callerDesc.get, callerDesc.set, "caller getter/setter should be same function");
+
+assert.sameValue(typeof argumentsDesc.get, "function");
+assert.sameValue(typeof argumentsDesc.set, "function");
+assert.sameValue(argumentsDesc.get, argumentsDesc.set, "arguments getter/setter should be same function");
```

Comment:
This is already covered by the existing tests in:
- https://github.com/tc39/test262/blob/main/test/built-ins/Function/prototype/restricted-property-arguments.js
- https://github.com/tc39/test262/blob/main/test/built-ins/Function/prototype/restricted-property-caller.js

Reply:
@gibson042 I added the refactored tests. For the other files I'm not sure what to do. Should I still address the issues or will they be removed?

Reply:
@gibson042 @anba Should I still address the other reviews/suggestions?